docs/how-to/use-chisel.rst

```diff
@@ -9,7 +9,27 @@ field without any ambiguities because slices follow an underscore-driven naming
 convention. For instance, ``openssl`` means the whole OpenSSL package, while
 ``openssl_bins`` means just the binaries slice of the OpenSSL package.
 Rockcraft will take care of the installation and priming of your
-content into the ROCK. There's an example :ref:`here <chisel-example>`.
+content into the ROCK.
+
+Example
+.......
+
+.. _chisel-example:
+
+.. code-block:: yaml
+
+  parts:
+    chisel-openssl:
+      plugin: nil
+      stage-packages:
+        - openssl_bins
+        - ca-certificates_data
+
+    package-hello:
+      plugin: nil
+      stage-packages:
+        - hello
```

Comment:
I know this is coming from another page, but given that this is a guide, we can't have untested code blocks. This needs to go into the `code` folder to be tested by Spread.

Reply:
After discussion, we decided to use the exists code for that example.

Reply:
ACK